lte/gateway/c/oai/lib/openflow/controller/ControllerEvents.cpp

```diff
@@ -173,14 +177,17 @@ AddGTPTunnelEvent::AddGTPTunnelEvent(
 AddGTPTunnelEvent::AddGTPTunnelEvent(
     const struct in_addr ue_ip, struct in6_addr* ue_ipv6, int vlan,
     const struct in_addr enb_ip, const struct in_addr pgw_ip,
-    const uint32_t in_tei, const uint32_t out_tei, const char* imsi,
+    const uint32_t in_tei, const uint32_t out_tei,
```

Comment:
to be consistent across the board, shall we change these as enb_in_tei and enb_out_tei? to be completely agnostic, might be better to replace 'enb' with 'node1' and pgs with 'node2' across the board.

Reply:
I would like to make naming change in separate PR. since we also need better names for the APIs.
Combining all these changes will make review harder.